test/gie/nkg.gie

```diff
@@ -0,0 +1,240 @@
+<gie>
```

Comment:
it would be better to use ``<gie-strict>`` mode for stronger syntax validation

Reply:
Done